logbook-logstash-logback-encoder/src/main/java/org/zalando/logbook/lle/LogstashLogbackSink.java

```diff
@@ -0,0 +1,49 @@
+package org.zalando.logbook.lle;
+
+import java.io.IOException;
+
+import org.slf4j.Marker;
+import org.zalando.logbook.Correlation;
+import org.zalando.logbook.HttpLogFormatter;
+import org.zalando.logbook.HttpRequest;
+import org.zalando.logbook.HttpResponse;
+import org.zalando.logbook.Precorrelation;
+import org.zalando.logbook.Sink;
+
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public final class LogstashLogbackSink implements Sink {
+
+    private final HttpLogFormatter formatter;
+    private final LogstashLogbackHttpLogWriter logWriter;
```

Comment:
I'd actually inline the logic form this one and drop the `LogstashLogbackHttpLogWriter`.